src/main/java/uk/gov/di/ipv/cri/common/library/service/ConfigurationService.java

```diff
@@ -20,6 +20,7 @@ public class ConfigurationService {
     private static final String PARAMETER_NAME_FORMAT = "/%s/%s";
     private static final long DEFAULT_BEARER_TOKEN_TTL_IN_SECS = 3600L;
     private static final Long AUTHORIZATION_CODE_TTL_IN_SECS = 600L;
+    public static final String CONFIG_SERVICE_CACHE_TTL_MINS = "CONFIG_SERVICE_CACHE_TTL_MINS";
```

Comment:
Given that the `CONFIG_SERVICE_CACHE_TTL_MINS` is only public due to it being consumed by the unit test class then it makes sense to make this field `private` and then simply duplicate in the unit test.

Reply:
I think the better long term solution is to collect all our env vars into an enum - but I'd still like to have one source of truth for that env var value.

Is a compromise to make the field `protected`?

Reply:
Let's just address this in a future PR, so that we can observe the benefit of the caching in x-ray etc